db/migrate/20220323122206_add_open_answer_to_poll_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddOpenAnswerToPollAnswers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :poll_answers, :open_answer, :string
```

Comment:
Since we're using `:string`, maybe we should add a length validation as well, so we don't get an exception when people answer with many suggestions :thinking:.

Another option would be using a `:text` field.